pudb/var_view.py

```diff
@@ -364,21 +500,107 @@ def get_stringifier(iinfo):
 # {{{ tree walking
 
 class ValueWalker:
+    BASIC_TYPES = []
+    BASIC_TYPES.append(type(None))
+    BASIC_TYPES.extend(integer_types)
+    BASIC_TYPES.extend(string_types)
+    BASIC_TYPES.extend((float, complex))
+    BASIC_TYPES = tuple(BASIC_TYPES)
+
+    NUM_PREVIEW_ITEMS = 3
+    MAX_PREVIEW_ITEM_LEN = 16
+
+    CONTENTS_LABEL = "<contents>"
```

Comment:
You're absolutely right, I have noticed this myself- it often doubles the number of key presses before I see the thing I want to look at. I'll experiment with having the `<contents>` item start off expanded by default.

The downside to this approach is if a user is actually trying to look at the attributes of a container-like item. I suspect that's not the most common use case, and it's easy enough to collapse the `<contents>` item when you do want to do this. Another thing I might try implementing soon is to allow a "count" before a motion command, similar to vim, so that you can jump around more precisely (e.g. to skip over the contents to the attributes without having to collapse the contents, which I find myself wanting to do sometimes).